Makefile

```diff
@@ -29,7 +29,7 @@ upgrade:       										## Upgrade all dependencies to the latest stable versio
 
 .PHONY: install
 install:
-	@uv sync --python 3.12
+	@uv sync
```

Comment:
might want to add `--frozen` so people don't inadvertently update the lockfile, but i'm not 100% sure so left that alone